test/rules/match-exported.js

```diff
@@ -135,3 +135,94 @@ ruleTester.run("lib/rules/match-exported", exportedRule, {
         }
     ]
 });
+
+var camelCaseCommonJS = "module.exports = variableName;",
```

Comment:
These should be at the top of the file (I probably should update the projects linting rules :wink: )